Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8.5
+FROM python:3.8

```

Comment:
hmmm,  we've been controlling the full version here.  do we know what version this will use?

Reply:
It will be 3.8.7 right now but I thought it will be better this way so that the fix versions are updated whenever we rebuild the image. Or do you think it is better to keep it fixed?

Reply:
unfortunately we have a whole big thing to follow when changing the version https://keboola.atlassian.net/wiki/spaces/SUP/pages/72024277/Python+R+Julia+Updates

Reply:
so, imho it's better left to a provoz task (unless this library requires 3.8.7?

Reply:
last point about it is that it's important to use the full version because even minor releases have a tendency to break things :-(

Reply:
omg, it is necessary also for fix (patch) versions? That's very scary field if updating to a patch version breaks some libraries or clients code. 😅 Ok, I'll keep there 3.8.5

Reply:
yeah, it's been a pain point for a long time

Reply:
ok, removed